ARS_simulation.py

```diff
@@ -300,8 +300,10 @@ async def simulate_workload_using_predictive_model(function: str, use_await=Fals
         startedCommands.pop(tid)
 
     with pr_lock:
-        for cmd in startedCommands.values():
-            cmd["parallelCommandsFinished"] = cmd["parallelCommandsFinished"] + 1
+        for key in startedCommands:
+            temp = startedCommands[key]
```

Comment:
die Variable temp darf ruhig cmd heißen :)
Kannst du Dir eigentlich gleich angewöhnen: Nenne niemals eine Variable `temp`. Kommt nie gut. Sobald du temp schreibst, sollte ein Reflex ausgelöst werden, der dir sofort auf die Finger klopft :D.

Reply:
wird in der nächsten Version geändert :D